.travis.yml

```diff
@@ -39,6 +40,7 @@ matrix:
 
   allow_failures:
     - php: hhvm
+    - php: 7.1
```

Comment:
I doubt it's an "actual" failure. The php 5.6 job for mysql is also consistently failing. But nvm, we can always update later.